src/styles/blocks/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  background: $main-color;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 48px 46px 0;
```

Comment:
<img width="899" alt="image" src="https://user-images.githubusercontent.com/102411130/231377602-e95eda62-1084-4f65-9ff6-4d6957656a19.png">

Reply:
do not forget about css properties order
http://codeguide.academy/html-css.html#css-order